src/main/java/uk/gov/hmcts/reform/laubackend/cases/response/AccessRequestGetResponse.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.laubackend.cases.response;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.laubackend.cases.dto.AccessRequestLog;
+
+import java.util.List;
+
+@Getter
+@Setter
+@Builder
+public class AccessRequestGetResponse {
+
+    private List<AccessRequestLog> accessLog;
+
+    private int startRecordNumber;
+
+    private boolean moreRecords;
+
+    private long totalNumberOfRecords;
```

Comment:
Total number of records was not there in LLD , If returning we need to include in LLD as we are doing for other end points

Reply:
Yes, it wasn't there, but I decided to add it in order to be consistent with the other GET endpoints.